regulations/templates/regulations/toc-preamble.html

```diff
@@ -1,14 +1,18 @@
-<ol>
-  {% for item in preamble_toc %}
-  <li>
-    <a
-        href="{{item.url}}"
-        data-section-id="{{item.full_id}}">
-      {{item.title}}
-    </a>
-    {% with preamble_toc=item.children %}
-      {% include "regulations/toc-preamble.html" %}
-    {% endwith %}
-  </li>
-  {% endfor %}
-</ol>
+{% if preamble_toc %}
```

Comment:
I should probably throw a comment in there to explain but that is so an
empty <ol> doesn't get produced when it loops back through and there is a
section header with no content inside. Because the empty <ol> it gets
styled which makes for random border lines of double thickness.

On Tue, Apr 19, 2016 at 3:19 PM, CM Lubinski <notifications@github.com>
wrote:

> In regulations/templates/regulations/toc-preamble.html
> <https://github.com/eregs/regulations-site/pull/257#discussion_r60303561>:
>
> > @@ -1,14 +1,18 @@
> > -<ol>
> > -  {% for item in preamble_toc %}
> > -  <li>
> > -    <a
> > -        href="{{item.url}}"
> > -        data-section-id="{{item.full_id}}">
> > -      {{item.title}}
> > -    </a>
> > -    {% with preamble_toc=item.children %}
> > -      {% include "regulations/toc-preamble.html" %}
> > -    {% endwith %}
> > -  </li>
> > -  {% endfor %}
> > -</ol>
> > +{% if preamble_toc %}
>
> Quick check, shouldn't preamble_toc always be present?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/eregs/regulations-site/pull/257/files/912c242adf3b83c7c36f9fee65268acf203ea505#r60303561>
>



-- 
John Donmoyer
18F – User Experience Designer
312-869-0339